src/backend/cc.h

```diff
@@ -760,12 +770,15 @@ struct func_t
     size_t typesTableDim;       // number used in typesTable[]
     size_t typesTableCapacity;  // allocated capacity of typesTable[]
 
-#if ELFOBJ
-    unsigned LSDAoffset;        // offset in LSDA segment of the LSDA data for this function
-#endif
-#if MACHOBJ
-    Symbol *LSDAsym;            // GCC_except_table%d
-#endif
+    union
+    {
+        unsigned LSDAoffset_;   // ELFOBJ: offset in LSDA segment of the LSDA data for this function
+        #define LSDAoffset lsda.LSDAoffset_;
+        Symbol *LSDAsym_;       // MACHOBJ: GCC_except_table%d
+        #define LSDAsym lsda.LSDsym_;
```

Comment:
It seems that the build is failing because of the macros that end in `;`. Can you make the union anonymous and remove the macros?